catchup/service.go

```diff
@@ -310,7 +310,8 @@ func (s *Service) fetchAndWrite(ctx context.Context, r basics.Round, prevFetchCo
 				return false
 			}
 			failureRank := peerRankDownloadFailed
-			if err == errNoBlockForRound {
+			var nbfe noBlockForRoundError
+			if errors.As(err, &nbfe) {
```

Comment:
This looks like it was meant for a different PR

Reply:
I guess these both fix master but yes, has different cause